src/test/server/Server_test.cpp

```diff
@@ -289,22 +284,22 @@ class Server_test : public beast::unit_test::suite
             thread.get_io_service(), journal);
         std::vector<Port> list;
         list.resize(1);
```

Comment:
I'd get rid of the `list.resize(1)` and change the declaration to `std::vector<Port> list{1};`. I also don't love the vector is named `list`. (I know this isn't in your change set, just noticing while reviewing the code)